test/tests/pbs_smoketest.py

```diff
@@ -147,20 +147,20 @@ def test_standing_reservation_stale_nodes(self):
              'reserve_end': time_now + 14,
              'Resource_List.place': 'free'}
         r = Reservation(TEST_USER, a)
-        #Submitting recurring reservation to check the resv status
-        #for one of the vnode being stale
+        # Submitting recurring reservation to check the resv status
+        # for one of the vnode being stale
         rid = self.server.submit(r)
-        #To check the standing resv once confirmed.
+        # To check the standing resv once confirmed.
         a = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2")}
-        #To check the standing resv for Degraded status
-        #when one of the vnode becomes down
+        # To check the standing resv for Degraded status
+        # when one of the vnode becomes down
         a2 = {'reserve_state': (MATCH_RE, "RESV_DEGRADED|10")}
         self.server.expect(RESV, a, id=rid)
         self.server.status(RESV, 'resv_nodes', id=rid)
         resv_node = self.server.reservations[rid].get_vnodes()[0]
-        #To check the status of standing resv in next cycle and also
-        #check the down node doesn't exist in resv_nodes.
-        a = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2"),
+        # To check the status of standing resv in next cycle and also
+        # check the down node doesn't exist in resv_nodes.
+        a = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2|RESV_RUNNING|5"),
```

Comment:
You should know the state you are looking for.  Why are you checking for confirmed or running?  When you pass this to expect(), you'll need to pass attrop=PTL_AND to check for both of these.

Reply:
I don't think we need check for both RUNNING and CONFIRMED, just CONFIRMED is enough.  Looks like Anamika has suggested to check if it's in race condition, when test case is waiting and it skipped the 8 seconds time.

Reply:
Hi Bhroam, yes I suggested to look for both since reservation start time is 8s from now and in the past we have observed related race conditions where reservation has started running by the time we are looking for reservation state. Also it should be PTL_OR as state will be one of them.

Reply:
Oh, OK.  It's OK then.  I don't think it should be PTL_OR.  That is an or between attributes.  He has correctly done the or as an regex.  It needs to be PTL_AND because there are two attributes he's expecting.  I don't think the regex he has will work for matching the resv_nodes though.

Reply:
I see. Sorry got confused about PTL_AND. you are right about it.